docs/rules/rules.tex

```diff
@@ -611,11 +618,13 @@ \section{Extending a chain}
 A block $b$ is said to be a valid extension of a valid chain
 $b_1,…,b_n$ if all of the following hold
 \begin{description}
-\item[Predecessor] The predecessor of $b$ is $b_n$ or, if $n=0$, the
-  predecessor of $b$ is the genesis identifier $b₀$.
+\item[Predecessor] The predecessor of $b$ is $b_n$ or, if $n=0$, the predecessor
+  of $b$ is the genesis identifier $b₀$. \improvement{This doesn't strictly make
+    sense, because we would be extending the chain $b_1, b_0$, but maybe we
+    don't care since it's obvious what the intention is.}
```

Comment:
It's habitual, in maths, to say that `b_1,…,b_0` is the empty list. It's also habitual in software. So this looks uncontroversial.